source/MailKitSimplified.Sender/Services/EmailWriter.cs

```diff
@@ -35,6 +35,9 @@ public EmailWriter(ISmtpSender emailClient, ILogger<EmailWriter> logger = null,
             _logger = logger ?? NullLogger<EmailWriter>.Instance;
             _emailClient = emailClient ?? throw new ArgumentNullException(nameof(emailClient));
             _fileSystem = fileSystem ?? new FileSystem();
+            _defaultFrom = emailClient.DefaultFrom != null && !string.IsNullOrEmpty(emailClient.DefaultFrom.Address)
+                ? emailClient.DefaultFrom
+                : null;
```

Comment:
Stylistically my preference is to do the null check in-line:
```cs
_defaultFrom = !string.IsNullOrEmpty(emailClient.DefaultFrom?.Address) ?
    emailClient.DefaultFrom : null;
```

Reply:
Yes, you are right, I like your way of writing too.